lib/fluent/plugin/out_s3.rb

```diff
@@ -216,7 +216,12 @@ def write(chunk)
         tmp.rewind
         log.debug { "out_s3: write chunk: {key:#{chunk.key},tsuffix:#{tsuffix(chunk)}} to s3://#{@s3_bucket}/#{s3path}" }
 
-        put_options = {:body => tmp, :content_type => @compressor.content_type, :storage_class => @storage_class}
+        put_options = {
+          :body => tmp,
+          :content_type => @compressor.content_type,
+          :storage_class => @storage_class,
+          :acl => @acl
```

Comment:
SDK doesn't provide default setting?
If SDK has default, we should not set `acl` when user doesn't specify `acl` parameter?